src/Infrastructure/Identity/RoleService.cs

```diff
@@ -201,16 +216,6 @@ public async Task<string> UpdatePermissionsAsync(string roleId, List<UpdatePermi
             }
```

Comment:
I would inline the AddPermissionClaimAsync extension method and eliminate it, as this is the only place where it's used... will make this cleaner IMO.

Reply:
And while we're at it... I don't really see the point of saving up the errors and then throwing them at the end. I would just immediately throw when there's a problem. The rolemanager.RemoveClaimAsync can also return a failed result btw... (there's probably more places where that isn't checked yet). I'm not really sure in which case that call - or AddClaimAsync for that matter - would fail though...